### PR TITLE
Update mysql-config.yaml

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #33223 

/language en
/sig docs storage
/kind cleanup

### CODE REVIEW

**Comment by Sea-n** [comment]
Could you explain why this PR is correct in the PR description? It's not a good practice to ask others go though an issue before reviewing.

And I don't understand why this PR should be labeled as `kind/cleanup`.

**Review by divya-mohan0209 — COMMENTED** [review]
Another reviewer/approver will need to approve this before it gets merged. But as-is, this change looks okay to me.
/lgtm

File: content/en/examples/application/mysql/mysql-configmap.yaml [hunk 1]
```diff
@@ -10,8 +10,9 @@ data:
     # Apply this config only on the primary.
     [mysqld]
     log-bin
+    # datadir=/var/lib/mysql/mysql
   replica.cnf: |
     # Apply this config only on replicas.
     [mysqld]
     super-read-only
-
+    # datadir=/var/lib/mysql/mysql
```
Comment (annajung) [inline content/en/examples/application/mysql/mysql-configmap.yaml:18]: I'm a bit confused about how this resolves the issue? aren't theses just comments? can you provide that running the steps with updated configmap modified resolved the original issue?

Comment (Kartik494) [reply content/en/examples/application/mysql/mysql-configmap.yaml:18]: Sure i will paste the steps here.

Comment (reylejano) [reply content/en/examples/application/mysql/mysql-configmap.yaml:18]: @Kartik494 , stilling working on this PR?
Will you paste steps here like mentioned [previously](https://github.com/kubernetes/website/pull/34694#discussion_r943086138)

Comment (Kartik494) [reply content/en/examples/application/mysql/mysql-configmap.yaml:18]: @reylejano i have been busy in doing other assignments, sure i will paste this soon.

**Comment by Sea-n** [comment]
Hey @Kartik494,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Pull request closed without merging**